test/verify/check-machines

```diff
@@ -148,10 +148,16 @@ class TestMachines(MachineCase):
         m.execute("sed -i 's/\"priority\".*$/\"priority\": 100,/' {0}".format("/usr/share/cockpit/machines/manifest.json"))
         m.execute("[ ! -e {0} ] || sed -i 's/\"priority\".*$/\"priority\": 0,/' {0}".format("/usr/share/cockpit/ovirt/manifest.json"))
 
+    def getLibvirtName(self):
+        name = self.machine.execute(
+            "systemctl list-unit-files | grep -o libvirt-bin.service || echo libvirtd.service").replace('\n', '')
+        return self.machine.execute(
+            "systemctl  --property=Id show {0} | cut -c 4-".format(name)).replace('\n', '')
+
     def startLibvirt(self):
         m = self.machine
         # Ensure everything has started correctly
-        m.execute("systemctl start libvirtd")
+        m.execute("systemctl start {0}".format(self.getLibvirtName()))
```

Comment:
The tests were passing prior this PR. What has changed that this is needed?

Reply:
it works, but disabling libvirt will make this stop working in ubuntus where `libvirt-bin.service` can have a priority. `libvirtd.service` is an alias when `libvirt-bin.service` is enabled